easybuild/easyblocks/n/namd.py

```diff
@@ -112,7 +132,7 @@ def configure_step(self):
                 self.cfg.update('namd_cfg_opts', "--with-fftw")
             self.cfg.update('namd_cfg_opts', "--fftw-prefix %s" % fftw)
 
-        namd_charm_arch = "--charm-arch %s" % '-'.join(self.cfg['charm_arch'].strip().split(' '))
+        namd_charm_arch = "--charm-arch %s" % '-'.join(self.cfg['charm_arch'].strip().split())
```

Comment:
but `charm_arch` may contain a space (see https://github.com/hpcugent/easybuild-easyconfigs/search?utf8=%E2%9C%93&q=charm_arch&type=), you're changing the `split`